app/SearchApi/SearchRequest.Adaptor/Publisher/Models/SearchRequestNotificationEvent.cs

```diff
@@ -40,5 +40,9 @@ public SearchRequestNotificationEvent()
         public NotificationType NotificationType { get; set; }
 
         public string Message { get; set; }
+
+        public string FSOName { get; set; }
+
+        public Person Person { get; set; }
```

Comment:
This is assuming we can only send on person to FMEP right ? Can we have multiple person ? Not sure though